client/src/components/App.jsx

```diff
@@ -58,29 +73,28 @@ class App extends React.Component {
   }
 
   componentDidMount() {
-    this.getRoomData(this.props.selectedRoom);
+    const id = window.location.pathname.split('/')[2];
+    this.getRoomData(id);
   }
 
   render () {
     const { error, isLoaded, items } = this.state;
 
     if (error) {
-      return <div>Error: {error.message}</div>;
+      return <Text>Error: {error.message}</Text>;
     } else if (!isLoaded) {
-      return <div> Loading... </div>
+      return <Text> Loading... </Text>
```

Comment:
@tegsheeb Just out of curiosity, where is the '< Text >' element coming from? Is this a styled-component or something else?

Reply:
Good one. I have deleted my Text styled component and it has same functionality as Body. I have refactored into new commit now.